### DISCUSSION TITLE
是否可以将V2ray转换为去中心化的一个工具？

### DISCUSSION THREAD

**Discussion by jingyuexing** [v2fly/v2ray-core#2018] [Ideas]
将每一个想要为别人代理的机器开启一个类似于矿工的功能,
根据代理过的流量进行发送代币,至于流量的私密性,当前的已经可以做到了了,对于流量的加密.
当我们需要代理机器,可以向别的机器支付流量代理的代币,支付成本

**Comment by scriptletn** [comment]
真要实现了，无非以下两种结果：
1、蜜罐满天飞。
2、投机家入行炒代币割韭菜，最后一地鸡毛。
